src/extension/utils/admin.js

```diff
@@ -75,14 +74,12 @@ export async function callAdmin(
     method = 'get',
     body = undefined,
     searchParams = new URLSearchParams(),
-    omitCredentials = false,
   } = {},
 ) {
   const url = createAdminUrl(config, api, path, searchParams);
   return fetch(url, {
     method,
     cache: 'no-store',
-    credentials: omitCredentials ? 'omit' : 'include',
```

Comment:
Should we not explicitly set it to `omit` instead of using the default (`same-origin`)?

Reply:
Sure, I think we can try it out. Since the sidekick doesn't work on `admin.hlx.page`, I think it should have the same end result, but it's definitely more certain.
I'll make the change and test it to see.

Reply:
fixed in [e44abed](https://github.com/adobe/aem-sidekick/pull/353/commits/e44abed15b685f3b4762dccfd4ae3427a6432b83) 👍